doc/rose-command.html

```diff
@@ -2196,6 +2196,10 @@ <h2 id="rose-suite-shutdown">rose suite-shutdown</h2>
 
       <dd>Switch off interactive prompting (=answer yes to everything).</dd>
 
+      <dt><kbd>-- --now, -- -n</kbd></dt>
```

Comment:
As discussed, not keen on you documenting cylc behaviour here. Instead, can you point to the cylc shutdown documentation along lines of "see cylc shutdown for options EXTRA-ARGS".